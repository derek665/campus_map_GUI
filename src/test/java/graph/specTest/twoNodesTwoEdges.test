# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and edges can be printed out.

# Create a graph
CreateGraph g1

# Add pair of nodes
AddNode g1 n1
AddNode g1 n2

# Add edges
AddEdge g1 n1 n2 e1
AddEdge g1 n2 n1 e2

# Print nodes of graph
ListNodes g1

# Print children of each node
ListChildren g1 n1
ListChildren g1 n2