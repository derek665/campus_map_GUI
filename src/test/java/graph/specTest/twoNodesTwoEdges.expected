# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and edges can be printed out.

# Create a graph
created graph g1

# Add pair of nodes
added node n1 to g1
added node n2 to g1

# Add edges
added edge e1 from n1 to n2 in g1
added edge e2 from n2 to n1 in g1

# Print nodes of graph
g1 contains: n1 n2

# Print edges of each node
the children of n1 in g1 are: n2(e1)
the children of n2 in g1 are: n1(e2)
